sound/soc/sof/ipc4-pcm.c

```diff
@@ -13,6 +13,33 @@
 #include "ipc4-priv.h"
 #include "ipc4-topology.h"
 
+static int sof_ipc4_set_multi_pipeline_state(struct snd_sof_dev *sdev, u32 state,
+					     struct ipc4_pipeline_set_state_data *data)
+{
+	struct sof_ipc4_msg msg = {{ 0 }};
+	u32 primary, ipc_size;
+
```

Comment:
Need to check ```count == 0```

Reply:
It is checked in sof_ipc4_set_pipeline_state()